File: 802.java
class Test
{
    public static void main(String args[])
    {
        int Arr[] = new int[6];

        Arr[0] = 2;
        Arr[1] = 8;
        Arr[2] = 7;
        Arr[3] = 5;
        Arr[4] = 2;
        Arr[5] = 1;

        int i = 0;
        int j = 0;
        int index = Arr.length;
        for(i = 0; i < Arr.length; i++)
        {
            for( j = i + 1; j < Arr.length; j++)
            {
                if(Arr[i] <= Arr[j])
                    break;

            }

            if(j == index )
                System.out.print(Arr[i] + " ");

        
                

            }
        }
            
        




    }

File: 802.cpp
#include<iostream>
using namespace std;

bool LinearSearch(int *Arr,int iSize,int iElement)
{
    bool bRet = false;
    int iCnt = 0;

    for(int i = 0; i < iSize; i++)
    {
        iCnt++;
        if(Arr[i] == iElement)
        {
            bRet = true;
            break;
        }

    }

    cout<<"Total iterations are : "<<iCnt<<"\n";

    return bRet;
}

int main()
{
    int Arr[] = {12,28,35,40,45,60};

    bool bRet = LinearSearch(Arr,6,60);

    if(bRet == false)
        cout<<"Not there in array\n";
    else
        cout<<"Present in array\n";
    return 0;
}